vispy/color/color_array.py

```diff
@@ -91,6 +91,10 @@ class ColorArray(object):
         If array-like, it must be an Nx3 or Nx4 array-like object.
         Can also be a list of colors, such as
         ``['red', '#00ff00', ColorArray('blue')]``.
+    color_space : "rgb" | "hsv"
+       if rgb, color tuples are interpreted as (r, g, b) components.
+       if hsv, color tuples are interpreted as (h, s, v) components.
+       if neither are specified, it is assumed to be "rgb" by default
```

Comment:
You can just say `(default : "rgb")`. Also, please capitalize the `if`'s at the start of the sentences.